src/main/java/beam/utils/VolumesAnalyzerFixed.java

```diff
@@ -49,4 +75,23 @@ public VolumesAnalyzerFixed(int timeBinSizeInSeconds, int maxTime, Network netwo
         return volumes;
     }
 
+    public double[] getVolumesPerHourForLink(final Id<Link> linkId, String mode) {
```

Comment:
This seems to not be used as far as I can tell?

Reply:
It is used BeamCalcLinkStats.java (line 75)